yt_dlp/extractor/bandlab.py

```diff
@@ -0,0 +1,438 @@
+    }, {
+        'url': 'https://www.bandlab.com/slytheband/collections/955102d4-1040-ef11-86c3-000d3a42581b',
+        'info_dict': {
+            'id': '955102d4-1040-ef11-86c3-000d3a42581b',
+            'timestamp': 1720762659,
+            'view_count': int,
+            'title': 'My Shit 🖤',
+            'uploader_id': 'slytheband',
+            'uploader': '𝓢𝓛𝓨',
+            'upload_date': '20240712',
+            'like_count': int,
+            'thumbnail': 'https://bandlabimages.azureedge.net/v1.0/collections/2c64ca12-b180-4b76-8587-7a8da76bddc8/',
+        },
+        'playlist_count': 15,
+    }, {
+        # Embeds can contain both albums and collections with the same URL pattern. This is an album
+        'url': 'https://www.bandlab.com/embed/collection/?id=12cc6f7f-951b-ee11-907c-00224844f303',
+        'info_dict': {
+            'id': '12cc6f7f-951b-ee11-907c-00224844f303',
+            'release_date': '20230706',
+            'description': 'This is a collection of songs I created when I had an Amiga computer.',
+            'view_count': int,
+            'title': 'Mark Salud The Amiga Collection',
+            'uploader_id': 'mssirmooth1962',
+            'comment_count': int,
+            'thumbnail': 'https://bl-prod-images.azureedge.net/v1.3/albums/d618bd7b-0537-40d5-bdd8-61b066e77d59/',
+            'like_count': int,
+            'uploader': 'Mark Salud',
+            'album': 'Mark Salud The Amiga Collection',
+            'album_type': 'LP',
+        },
+        'playlist_count': 24,
+    }, {
+        # Tracks without revision id
+        'url': 'https://www.bandlab.com/embed/collection/?id=e98aafb5-d932-ee11-b8f0-00224844c719',
+        'info_dict': {
+            'like_count': int,
+            'uploader_id': 'sorakime',
+            'comment_count': int,
+            'uploader': 'Sorakime',
+            'view_count': int,
+            'description': 'md5:4ec31c568a5f5a5a2b17572ea64c3825',
+            'release_date': '20230812',
+            'title': 'Art',
+            'album': 'Art',
+            'album_type': 'Album',
+            'id': 'e98aafb5-d932-ee11-b8f0-00224844c719',
+            'thumbnail': 'https://bl-prod-images.azureedge.net/v1.3/albums/20c890de-e94a-4422-828a-2da6377a13c8/',
+        },
+        'playlist_count': 13,
+    }, {
+        'url': 'https://www.bandlab.com/albums/89b79ea6-de42-ed11-b495-00224845aac7',
+        'only_matching': True,
+    }]
+
+    def _entries(self, album_data):
+        for post in traverse_obj(album_data, ('posts', lambda _, v: v['type'])):
+            post_type = post['type']
+            if post_type == 'Revision':
+                yield self._parse_revision(post.get('revision'))
+            elif post_type == 'Track':
+                yield self._parse_track(post)
+            elif post_type == 'Video':
+                yield self._parse_video(post)
+            else:
+                self.report_warning(f'Skipping unknown post type: {post_type!r}')
```

Comment:
I think double-quotes look better in user-facing output, but I realize this is über-pedantic; it's up to you
```suggestion
                self.report_warning(f'Skipping unknown post type: "{post_type}"')
```